word_check/index.html

```diff
@@ -9,84 +9,224 @@
 </head>
   <body>
     <h1>English</h1>
-    <div class="row g-3 align-items-center">
-        <div class="col-1">
-            <label for="car">Автомобіль</label>
-        </div>
-        <div class="col-auto">
-            <input type="text"  class="form-control " id="car" value="">
-          </div>  
-    </div>
-    <div class="row g-3 align-items-center">
-        <div class="col-1">
-            <label for="bus">Автобус</label>
-        </div>
-        <div class="col-auto">
-            <input type="text"  class="form-control " id="bus" value="">
-          </div>
-    </div>
-    <div class="row g-3 align-items-center">
-        <div class="col-1">
-            <label for="bike">Велосипед</label>
-        </div>
-        <div class="col-auto">
-            <input type="text"  class="form-control" id="bike" value="">
-          </div>
-    </div>
-    <button type="submit" class="btn btn-primary mb-3" onclick="validate()">Первірети</button>
-
-
-   
 
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-kenU1KFdBIe4zVF0s0G1M5b4hcpxyD9F7jL+jjXkk+Q2h455rYXK/7HAuoJl+0I4" crossorigin="anonymous"></script>
+    
     <script>
-        function validate() {
+        //массив
+        var array = [
+            { en: 'car', ua: 'авто'},
+            { en: 'bike', ua: 'мото'},
+            { en: 'bus', ua: 'автобус'}];
 
-            let car = 'car';
-            let bus = 'bus';
-            let bike = 'bike';
+        for (const [key, val] of Object.entries(array)){
+            let row1 = document.createElement('div');
```

Comment:
creating of rows should be not in `validate` function.
cold you please create a separate function which creates all rows?